CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Print promtail-friendly logs. See #37
```

Comment:
I'd like to have the link to the PR as well. In the vast majority of cases we'll be reading the Changelog from our computers and having the link will come in handy.